clustered/integration-test/src/test/java/org/ehcache/clustered/replication/BasicClusteredCacheOpsReplicationTest.java

```diff
@@ -77,11 +82,16 @@
   public Consistency cacheConsistency;
 
   @ClassRule
-  public static Cluster CLUSTER =
-      newCluster(2).in(new File("build/cluster")).withServiceFragment(RESOURCE_CONFIG).build();
+  public static final ParallelTestCluster CLUSTER = new ParallelTestCluster(newCluster(2).in(new File("build/cluster")).withServiceFragment(RESOURCE_CONFIG).build());
+  @Rule
+  public final TestRule watchdog = CLUSTER.watchdog();
+
+  @Rule
+  public final TestName testName = new TestName();
 
   @Before
   public void startServers() throws Exception {
+    System.out.println("In @Before method");
```

Comment:
do we want to keep that ?

Reply:
Nope... left over from my debugging. Will clean up!